app/views/registration/steps/_about.html.haml

```diff
@@ -15,6 +15,7 @@
         .small-12.columns
           .alert-box.info{ "ofn-inline-alert" => true, ng: { show: "visible" } }
             %h6{ "ng-bind" => "'enterprise_success' | t:{enterprise: enterprise.name}" }
+            %span {{'enterprise_registration_exit_message' | t}}
```

Comment:
Thanks for the lesson @mkllnk!
I followed what's written here: https://github.com/openfoodfoundation/openfoodnetwork/wiki/i18n#javascript-translations-prefer-angular-filter-over-global-t-function
From your comment I conclude, this rule described in this i18n wiki page is only for javascript and in this case, because it's a classic view as you say, I should use the t function. Correct?
Can you please confirm this is correct? (I can update the wiki with this info you shared here)

Reply:
Yep, you are right @luisramos0, I believe :trollface:

Reply:
Exactly. The Angular filter is only for pre-compiled Javascript templates. Please use Rails' `t` function here.

Reply:
Hi there, I was about to do this but I realized that I did it this way because this is how it is done in all the registration pages. I just followed the local dialect. I dont want to fix only this one and leave everything else as it is, and different. Can we define the best way to do this and I'll fix in all pages of the registration process?

In the registration process, we are currently using one of two ways to translate, both javascript:
A - {{'registration_enterprise_address' | t}} - js but without namespace on the translation key
B - {{'enterprise.registration.modal.steps.type.question' | t}} - this way we can organize the translations file as well

@mkllnk, the suggestion you made is a third option, using the rails t function.
So, shall I change all registration process translation keys to be called using rails t function?

Reply:
Fair enough. It's always good to go with the local dialect. :-)

It would be awesome if you wouldn't mind changing all those messages to my option C organising everything using the lazy lookup.